frontend/src/metabase/querying/notebook/components/NotebookStep/NotebookActionButton/NotebookActionButton.module.css

```diff
@@ -13,11 +13,7 @@
     background-color: var(--mb-color-notebook-step-bg);
 
     &:hover {
-      background-color: color-mix(
-        in srgb,
-        var(--mb-color-notebook-step-bg),
-        black 10%
-      );
+      background-color: var(--mb-color-notebook-step-bg-hover);
```

Comment:
Uses the dynamic variable so we handle dark and light mode hovers using different color transformations.